.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+### Pull Request Description
+
+<!-- link to the k/enhancements issue -->
+- Issue link:
+
+<!-- short description of work done in PR e.g. updating milestone, adding new KEP, adding test requirements… -->  
+- One-line PR description:
```

Comment:
Maybe swap the order of these two? Describe the change first and then link the k/e issue?

Also, can we explicitly mention not to use phrases like `fixes #NNNN` in the issue link field? In this repo, we don't want PR merges to close the corresponding k/e issues unless the Enhancement is graduating to stable/being deprecated.

Reply:
done! PTAL

Reply:
Thank you!